supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc

```diff
@@ -149,128 +143,85 @@ Use "Memory Overcommitment Manager (MOM)" for the first instance in a section, a
-
-[discrete]
-[[red-hat-enterprise-virtualization-hypervisor]]
-==== image:images/yes.png[yes] Red Hat Enterprise Virtualization Hypervisor (noun)
-*Description*: Red Hat Enterprise Virtualization Hypervisor is one of the types of host in Red Hat Enterprise Virtualization (3.x). It is a minimal operating system based on Red Hat Enterprise Linux, is distributed as an ISO file, and is a closed system. File system access and root access are limited. `yum` is disabled.
-
-Use "Red Hat Enterprise Virtualization Hypervisor (RHEV-H)" for the first instance in a section. "RHEV-H" can be used for subsequent instances. It can also be referred to as "the Hypervisor", as long as the H is capitalized to avoid confusion with hypervisors in general. Do not use in Red Hat Virtualization 4.x.
+[[monitoring_portal]]
+==== image:images/yes.png[yes] Monitoring Portal (noun)
+*Description*: The Monitoring Portal is a graphical user interface used to display reports based on data collected from the Data Warehouse PostgreSQL database. The Monitoring Portal is implemented by using the Grafana web-based UI tool to display the reports as dashboards.
 
 *Use it*: yes
 
-*Incorrect forms*: RHEVH, Red Hat Enterprise Virtualization Host, RHEV Hypervisor
-
-*See also*: xref:host-rhv[host], xref:red-hat-virtualization-host[Red Hat Virtualization Host]
-
-[discrete]
-[[red-hat-enterprise-virtualization-manager]]
-==== image:images/yes.png[yes] Red Hat Enterprise Virtualization Manager (noun)
-*Description*: The Red Hat Enterprise Virtualization Manager is a server that manages and provides access to the resources in the Red Hat Enterprise Virtualization environment.
-
-Use "Red Hat Enterprise Virtualization Manager" for version 3.x. Spell out in full for the first instance in a section. Use "the Manager" for subsequent instances. Do not use "the engine", which is the oVirt (upstream) term.
-
-*Use it*: yes
-
-*Incorrect forms*: RHEVM, RHEV-M, RHEV Manager, engine
+*Incorrect forms*:
 
-*See also*: xref:red-hat-virtualization-manager[Red Hat Virtualization Manager]
+*See also*: xref:data-warehouse[Data Warehouse]
 
 [discrete]
-[[red-hat-enterprise-virtualization-manager-reports]]
-==== image:images/yes.png[yes] Red Hat Enterprise Virtualization Manager Reports (noun)
-*Description*: Red Hat Enterprise Virtualization Manager Reports is available as an optional component. It produces reports that can be built and accessed through a web user interface, and then rendered to screen, printed, or exported to a variety of formats.
-
-This component was removed from Red Hat Virtualization (4.x), but still exists in Red Hat Enterprise Virtualization (3.x).
+[[red-hat-enterprise-linux-host]]
+==== image:images/yes.png[yes] Red Hat Enterprise Linux host (noun)
+*Description*: Red Hat Enterprise Linux servers subscribed to the appropriate entitlements can be used as hosts in Red Hat Virtualization.
```

Comment:
This is a bit nitpicky but you could maybe make this sentence active and more user oriented with something like:

`You can use Red Hat Enterprise Linux servers that are subscribed to the appropriate entitlements as hosts in Red Hat Virtualization.`